lib/oxide-vpc/tests/common/mod.rs

```diff
@@ -95,6 +97,12 @@ pub const VPC_LAYERS: [&str; 5] =
 pub const GW_MAC_ADDR: MacAddr =
     MacAddr::from_const([0xA8, 0x40, 0x25, 0xFF, 0x77, 0x77]);
 
+pub const BS_MAC_ADDR: MacAddr =
+    MacAddr::from_const([0xA8, 0x40, 0x25, 0x77, 0x77, 0x77]);
+
+pub const BS_IP_ADDR: Ipv6Addr =
+    Ipv6Addr::from_const([0xfd00, 0x9900, 0, 0, 0, 0, 0, 1]);
```

Comment:
Again, this is just test-only code but for consistency
```suggestion
    Ipv6Addr::from_const([0xfd00, 0x99, 0, 0, 0, 0, 0, 1]);
```

Reply:
done